.travis.yml

```diff
@@ -23,11 +23,12 @@ env:
 - TOX_ENV=py26e2e
 - TOX_ENV=py27ssh
 - TOX_ENV=py26ssh
+- TOX_ENV=docs
 install:
   - pip install --upgrade pip
   - pip install --upgrade setuptools
   - pip install tox
 script:
   - pip --version
   - tox --version
-  - tox -e $TOX_ENV
+  - PYTEST_PROCESSES=1 tox -e $TOX_ENV
```

Comment:
-1, because actually the name of the xdist argument is `--numprocesses`. Actually, it makes sense: the way xdist works is exactly by creating boxed subprocesses.